src/js/views/patients/widgets/widgets.scss

```diff
@@ -7,14 +7,17 @@
 }
 
 .widgets__form-widget {
+  align-items: center;
+  display: flex;
+  max-width: 204px;
```

Comment:
So I dislike `max-width` generally, _but_ the `div` this is in has a `max-width` of `256px`.  That said I think we probably have a problem there.  If something does have max-width of 256 we have inner horizontal scrolling.. we need to figure out where that went wrong before we can apply another max width.. which hopefully we don't need.